api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java

```diff
@@ -86,12 +86,12 @@ public CohortDefinition getNumberOfPatientsWhoInitiatedPreTarvByEndOfPreviousMon
         "A1III",
         map(
             getAllPatientsEnrolledInPreArtProgramWithDateEnrolledLessThanStartDateA1(),
-            "startDate=${startDate},location=${location}"));
+            "startDate=${startDate-1},location=${location}"));
```

Comment:
Same here; date parameters need to well done

Reply:
Updated as well